### PR TITLE
Replace nested scatter in TinyResolve with sharding subworkflow

### PR DESCRIPTION
**Pull request by epiercehoffman**
### Updates
Since nested WDL scatters can cause Cromwell issues, replaced nested scatter in `GatherBatchEvidence.TinyResolve` with a task to perform the sharding.

### Testing
* Validated all WDLs and JSONs with womtool
* Successfully ran `GatherBatchEvidence.wdl` with 105 samples and verified that the first four shards had 25 samples and the final shard had 5 samples based on the outputs from the sharding task. Also spot-checked the script for the resolve task for one shard and verified that it had the correct number of samples.

### CODE REVIEW

**Comment by epiercehoffman** [comment]
Tested subworkflow version:
* Validated with womtool
* Ran GatherBatchEvidence and verified TinyResolve completed successfully and checked number of samples per shard for final 3 shards

File: wdl/TinyResolve.wdl [hunk 1]
```diff
@@ -24,29 +28,47 @@ workflow TinyResolve {
...
-        samples=shard_samples,
+        raw_vcfs=GetShardVcfs.shard_items,
+        samples=GetShardSamples.shard_items,
         sv_pipeline_docker = sv_pipeline_docker,
         cytoband=cytoband,
         cytoband_idx=cytoband_idx,
-        discfile=shard_discfiles,
-        discfile_idx=shard_discfiles_idx,
+        discfile=GetShardDiscfiles.shard_items,
+        discfile_idx=GetShardDiscfileIndexes.shard_items,
```
Comment (mwalker174) [inline wdl/TinyResolve.wdl:71]: `discfile_idx=GetShardDiscfileIndexes.shard_items` -> `discfile_idx=GetShardDiscfiles.shard_items + ".tbi"`

File: wdl/GatherBatchEvidence.wdl [hunk 2]
```diff
@@ -170,6 +170,7 @@ workflow GatherBatchEvidence {
     String gatk_docker
     String? gcnv_gatk_docker
     String cnmops_docker
+    String? python_docker
```
Comment (mwalker174) [inline wdl/GatherBatchEvidence.wdl:173]: Is this needed?

File: wdl/TinyResolve.wdl (hunk 1 above)
Comment (mwalker174) [reply wdl/TinyResolve.wdl:69]: Ah yes won't work on arrays - ignore!

**Pull request merged**